index.html

```diff
@@ -205,43 +232,45 @@ <h1>About
                     <div class="let">LET’S CONNECT</div>
                     <div class="Self-icons">
                         <ul>
-                            <li><a href="#"><img src="/photos/github.jpg" alt=""></a></li>
-                            <li><a href="#"><img src="/photos/Linkedin-icon.jpg" alt=""></a></li>
-                            <li><a href="#"><img src="/photos/angellist.jpg" alt=""></a></li>
-                            <li><a href="#"><img src="/photos/twitter.jpg" alt=""></a></li>
-                            <li><a href="#"><img src="/photos/mail.jpg" alt=""></a></li>
+                            <li><a href="#"><img src="./photos/github.jpg" alt=""></a></li>
+                            <li><a href="#"><img src="./photos/Linkedin-icon.jpg" alt=""></a></li>
+                            <li><a href="#"><img src="./photos/angellist.jpg" alt=""></a></li>
+                            <li><a href="#"><img src="./photos/twitter.jpg" alt=""></a></li>
+                            <li><a href="#"><img src="./photos/mail.jpg" alt=""></a></li>
                         </ul>
                     </div>
                 </div>
-                <footer class="submit">
-                    <button type="submit">Get my resume</button>
-                </footer>
+                <div class="submit resume">
+                    <button  type="submit">Get my resume</button>
+                </div>
```

Comment:
- [ Optional Changes ] I see that when you were referencing the Figma design, there was a border around the button. That was because other students or viewers were hovering on that button and Figma made that border appear, but it's not meant to be in the original design. Please remove the border on the div that wraps the button to make your design look great 👍 

**Your Design**
![image](https://user-images.githubusercontent.com/50941074/167250585-833fa745-1297-494d-9bd5-b3ebb1391d71.png)

**You see, there is no border wrapping the button here on the Figma design**
![image](https://user-images.githubusercontent.com/50941074/167250620-f3745bfc-4086-4f7e-9dde-e08f46356513.png)